.github/workflows/pull-request.yml

```
@@ -195,14 +195,16 @@
         id: changed-files
         uses: tj-actions/changed-files@v45
 
-      - name: Set up Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: 3.9
+      - name: Filter TypeScript files
+        id: filter-files
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)